web/client/plugins/Identify.jsx

```diff
@@ -194,12 +193,22 @@ const IdentifyPlugin = compose(
         showRevGeocode: showMapinfoRevGeocode,
         hideRevGeocode: hideMapinfoRevGeocode,
         onEnableCenterToMarker: updateCenterToMarker.bind(null, 'enabled'),
-        onEdit: editLayerFeatures
+        onEdit: editLayerFeatures,
+        setDefaultIdentify: setDefaultIdentify
     }, (stateProps, dispatchProps, ownProps) => ({
         ...ownProps,
         ...stateProps,
         ...dispatchProps,
-        enabled: stateProps.enabled && (stateProps.isCesium || !ownProps.showInMapPopup) && !stateProps.floatingIdentifyEnabled
+        enabled: (
+            isNil(ownProps.enabled)
+            && isNil((stateProps.enabled && (stateProps.isCesium || !ownProps.showInMapPopup) && !stateProps.floatingIdentifyEnabled))
+            && true
+        )
+        || (stateProps.enabled
+            && (stateProps.isCesium
+            || !ownProps.showInMapPopup)
+            && !stateProps.floatingIdentifyEnabled)
+        || isNil(stateProps.enabled) && ownProps.enabled
```

Comment:
This is hard to undestand for me. I think it is because of the issues you encountered with enabled flag (see my other comments), so with the proposed solution, you can remove it